### PR TITLE
add enabled mode to segment specs

### PR DESCRIPTION
**Pull request by MarianoTucat**
this field will include info of the special rules that applies to this mode given the user's current bundle, if any

### CODE REVIEW

File: docs/specs/tripgo.swagger.yaml [hunk 1]
```diff
@@ -3850,3 +3853,19 @@ definitions:
       exchange:
         description: Rate to convert cost from user's local currency to dollar (optional).
         type: number
+
+  EnabledMode:
+    description: MaaS bundle enabled mode for this segment
+    properties:
+      mode:
+        type: string
+        description: mode of transport
```
Comment (nighthawk) [inline docs/specs/tripgo.swagger.yaml]: @MarianoTucat Out of curiosity: why is this needed? A segment already has a mode. This just doubles up on that, right? Can this ever be different?

File: docs/specs/tripgo.swagger.yaml [hunk 2]
```diff
@@ -3850,3 +3853,19 @@ definitions:
...
+      pointsPerCost:
+        type: number
+        description: 0 for unlimited/free, 1 by default, 0.9 for 10% discount
+      discountPoints:
+        type: number
+        description: in cents, 500 for 5 AUD fixed discount, not included if no discount
```
Comment (nighthawk) [inline docs/specs/tripgo.swagger.yaml]: I'm confused. Why is this in cents? Shouldn't it be in points? What if someone wants 1 point per dollar, or 10 points per dollar?

File: docs/specs/tripgo.swagger.yaml (hunk 1 above)
Comment (MarianoTucat) [reply docs/specs/tripgo.swagger.yaml]: This is not needed, we can remove it. Just adding it here because we are actually returning it.

File: docs/specs/tripgo.swagger.yaml (hunk 2 above)
Comment (MarianoTucat) [reply docs/specs/tripgo.swagger.yaml]: agree we need better wording here, just tried to make it clear on how we handle it in the BE.
